hi-ml/src/health_ml/lightning_container.py

```diff
@@ -237,6 +237,11 @@ def on_run_extra_validation_epoch(self) -> None:
             logging.warning("Hook `on_run_extra_validation_epoch` is not implemented by lightning module."
                             "The extra validation epoch won't produce any extra outputs.")
 
+    def ignore_pl_warnings(self) -> None:
```

Comment:
I think that's a bit too specialized. No problem if that is in BaseMIL, but I would not place it here.